config/packages/twig_extensions.yaml

```diff
@@ -1,10 +0,0 @@
-services:
-    _defaults:
-        public: false
-        autowire: true
-        autoconfigure: true
-
-    #Twig\Extensions\ArrayExtension: ~
-    #Twig\Extensions\DateExtension: ~
-    Twig\Extensions\IntlExtension: ~   # needed for the 'localizeddate' filter
-    #Twig\Extensions\TextExtension: ~
```

Comment:
Twig Extension is deprecated https://github.com/twigphp/Twig-extensions#twig-extensions-repository